qiime2/sdk/util.py

```diff
@@ -92,3 +92,25 @@ def parse_format(format_str):
     except KeyError:
         raise TypeError("No format: %s" % format_str)
     return format_record.format
+
+
+def has_metadata(artifact):
+    """ Checks for metadata within an artifact
+
+    Parameters
+    ----------
+    artifact: Artifact
+       The artifact to check for metadata.
+
+    Returns
+    -------
+    bool
+       True if the artifact has metadata, False otherwise.
+
+    """
+    from qiime2 import Metadata
+    from qiime2.core import transform
```

Comment:
change to fully qualified imports as well. (We tend to avoid `from` in the framework)